contrib/fedora/cjdns.dyn.patch

```diff
@@ -0,0 +1,57 @@
+diff -up ./node_build/make.js.dyn ./node_build/make.js
+--- ./node_build/make.js.dyn	2015-11-02 17:59:41.000000000 -0500
++++ ./node_build/make.js	2015-11-04 19:57:49.961155943 -0500
+@@ -238,41 +238,9 @@ Builder.configure({
```

Comment:
This patch will be a disaster to maintain, stuff will change which makes it fail to apply.